.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,29 @@
+name: "Test"
+
+on: push
+
+permissions:
+  contents: read
+
+jobs:
+  chromatic:
+    name: Run Chromatic
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22.x
+      - uses: pnpm/action-setup@v2
+        with:
+          version: latest
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
```

Comment:
You can copy these steps from [our `lint-test` workflow](https://github.com/Amsterdam/design-system/blob/develop/.github/workflows/lint-test.yml#L15), they do the same thing.